extensions/vscode/src/views/homeView.ts

```diff
@@ -120,49 +120,43 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     });
 
     useBus().on("activeConfigChanged", (cfg: Configuration | undefined) => {
-      const activeContentRecord = this.getActiveContentRecord();
-      if (!activeContentRecord) {
-        return;
-      }
```

Comment:
Instead of failing early we can make everything below this optional like it was previously. Using `undefined` and adapting to it in the sending, refreshing, and config watcher setup.